CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Changes are grouped as follows
 
 ### Added
 - The files API can now create a file without uploading content.
+- New method create_hierarchy() added to assets API.
```

Comment:
Under ### Changed, add that assets.create() no longer will validate and resolve asset hierarchy locally and post assets in correct order, and that that functionality has been replaced by assets.create_hierarchy(). 

We should probably wait with merging this PR until we have merged in the other PRs currently waiting and made a patch release. Then we bump the minor version for this change since it is a breaking change. 

Breaking changes would really mean that we should bump the major version. But since it is a very small change and bumping major would mean maintaining 2 major versions and porting bugfixes to 1.x, let's just bump the minor version and hope that no one gets too upset about it.